src/sshping.cxx

```diff
@@ -346,7 +442,7 @@ ssh_session begin_session() {
     }
 
     // Try to connect
-    clock_gettime(CLOCK_MONOTONIC, &t0);
+	t0 = GetTime();
```

Comment:
CamelCase - yuck!  It's always good to follow the style conventions already in use in the existing code.  Suggest `gettime()` or `get_time()`.